### PR TITLE
add source-map-support to webpack config

### PR DESCRIPTION
**Pull request by bkniffler**
https://github.com/NYTimes/kyt/issues/397

### CODE REVIEW

File: packages/kyt-core/config/webpack.dev.server.js [hunk 1]
```diff
@@ -53,5 +53,6 @@ module.exports = options => ({
   plugins: [
     new webpack.NoErrorsPlugin(),
     new webpack.optimize.LimitChunkCountPlugin({ maxChunks: 1 }),
+    new webpack.BannerPlugin('require("source-map-support").install();', { raw: true, entryOnly: false }),
```
Comment (delambo) [inline packages/kyt-core/config/webpack.dev.server.js]: I think you need to use the one option argument: 

`{ banner: 'require("source-map-support").install();', raw: true, entryOnly: false }`

**Comment by bkniffler** [comment]
Okay, done. I've also set `entryOnly: true` which should be sufficient.

**Comment by delambo** [comment]
Thanks for the help @bkniffler.

**Comment by zjye** [comment]
@bkniffler , in this case, will need to bring over the 'source-map-support' in node_modules together with the bundled js when deploy to server? or else it can't find the module when running with node

**Pull request merged**